docs/rst/amldashboard/user_manual/interfaces.rst

```diff
@@ -0,0 +1,141 @@
+.. include:: /rst/exports/alias.include
+.. include:: /rst/exports/roles.include
+
+.. _amldashboard_interfaces:
+
+##########
+Interfaces
+##########
```

Comment:
I will add a reference to the usage subsection where the images for each interface are explained in detail. Including the images here might be redundant since the usage section already covers this comprehensively.